Model/Ui/CapabilitiesConfigProvider.php

```diff
@@ -38,7 +38,8 @@ public function getConfig()
         foreach ($listOfActivePaymentMethods as $method) {
             if ($method->getCode() === OmiseInstallmentConfig::CODE) {
                 return [ 
-                    'installment_backends' => $this->capabilities->retrieveInstallmentBackends()
+                    'installment_backends' => $this->capabilities->retrieveInstallmentBackends(),
+                    'is_zero_interest' => $this->capabilities->isZeroInterest()
```

Comment:
Saw that we have to do such tasks to manipulate the list of instalment data at JS file. 
Wouldn't it be better to finish those tasks here before returning it out to frontend? Wonder what is the idea behind this approach (to manipulate list data at the client side).

Reply:
That's interesting. Do you mind to share me some resource or article that explains about a benefit of moving the data manipulation from server side to client side? Can be some later time, just want to study on this.

mark as resolved for now.